code/obj/item/device/pda2/base_os.dm

```diff
@@ -197,7 +196,7 @@
 						. += "<br>"
 
 					else
-						. += {"<a href='byond://?src=\ref[src];input=mailgroup'>Join/create group</a> | 
+						. += {"<a href='byond://?src=\ref[src];input=mailgroup'>Join/create group</a> |
```

Comment:
ditto

Reply:
fixed. no idea why vsccode is dumb and ignores my settings